src/tir/ir/stmt.cc

```diff
@@ -366,19 +366,19 @@ Allocate::Allocate(Var buffer_var, DataType dtype, Array<PrimExpr> extents, Prim
   data_ = std::move(node);
 }
 
-int32_t AllocateNode::constant_allocation_size(const Array<PrimExpr>& extents) {
-  int64_t result = 1;
+size_t AllocateNode::ConstantAllocationSize(const Array<PrimExpr>& extents) {
+  size_t result = 1;
```

Comment:
i think we should use the same data type as would be used to compute extents from a DLTensor, which i think is `int64_t`. thoughts?

Reply:
Changed to int64_t